taxi/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):

```

Comment:
According to the checklist, the `Meta` class should be implemented after all attributes are defined. However, in your `Driver` model, the `Meta` class is implemented before the `__str__` method. Please, move it after the `__str__` method.